opal/mca/btl/portals4/btl_portals4_component.c

```diff
@@ -658,15 +658,19 @@ mca_btl_portals4_component_progress(void)
 
                 tag = (unsigned char) (ev.hdr_data);
 
-                btl_base_descriptor.des_segments = seg;
-                btl_base_descriptor.des_segment_count = 1;
                 seg[0].seg_addr.pval = ev.start;
                 seg[0].seg_len = ev.mlength;
 
                 reg = mca_btl_base_active_message_trigger + tag;
+
+                /* TODO: retreive the endpoint if possible */
```

Comment:
typo in the comment. And a TODO loft.

Reply:
Deleting the TODO. Will leave it up to the component maintainer to add the endpoint if they want it. The btl base atomics code will lookup the endpoint if it is not supplied so it will work with this component but not optimally.